app/controllers/global_issue_templates_controller.rb

```diff
@@ -21,21 +21,20 @@ def index
         @template_map[Tracker.find(tracker.id)] = templates
       end
     end
-    render :layout => !request.xhr?
+    render layout: !request.xhr?
   end
 
   def new
     # create empty instance
     @trackers = Tracker.all
     @projects = Project.all
-    @global_issue_template = GlobalIssueTemplate.new(:author => @user,
-                                        :tracker => @tracker)
+    @global_issue_template = GlobalIssueTemplate.new(author: @user, tracker: @tracker)
     if request.post?
       # Case post, set attributes passed as parameters.
       @global_issue_template.safe_attributes = params[:global_issue_template]
       if @global_issue_template.save
         flash[:notice] = l(:notice_successful_create)
-        redirect_to :action => "show", :id => @global_issue_template.id
+        redirect_to action: "show", id: @global_issue_template.id
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)